src/wp-content/themes/twentytwentyfour/patterns/template-home-portfolio.php

```diff
@@ -10,7 +10,13 @@
 
 <!-- wp:template-part {"slug":"header"} /-->
 
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"0","bottom":"0","left":"0","right":"0"},"blockGap":"0","margin":{"top":"0","bottom":"0"}}},"layout":{"type":"default"},"tagName":"main"} -->
+<main class="wp-block-group" style="margin-top:0;margin-bottom:0;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0">
```

Comment:
```suggestion
<main class="wp-block-group" style="margin-top:0;margin-bottom:0;padding:0;">
```
Could we simplify this to use only `padding`?

Reply:
sadly that is not valid block markup